### PR TITLE
Update 'file-not-found'

### PR DESCRIPTION
**Pull request by v-pegao**
| Log Code | Changes |
|----------|---------------|
| file-not-found | message |

### CODE REVIEW

File: docs/specs/toc.yml [hunk 1]
```diff
@@ -814,7 +814,7 @@ inputs:
       tocHref: index.md
 outputs:
   build.log: |
-    ["warning","file-not-found","Cannot find file 'index.md' relative to 'docs/f1/TOC.yml'","docs/f1/TOC.yml",2,9]
+    ["warning","file-not-found","Invalid file link: 'index.md'.","docs/f1/TOC.yml",2,9]
```
[the remark this answers is not in the document]
Comment (fenxu) [reply docs/specs/toc.yml]: Why does index.md path change to pathToRoot? Isn't it a relative path to this file?

---
In reply to: [282055459](https://github.com/dotnet/docfx/pull/4505#discussion_r282055459) [](ancestors = 282055459)

[the remark this answers is not in the document]
Comment (yufeih) [reply docs/specs/toc.yml]: That might not be obvious in the new message, @v-pegao , can you help confirm if it should be a link relative to the current file or a link relative to repository root?

[the remark this answers is not in the document]
Comment (fenxu) [reply docs/specs/toc.yml]: We can confirm with PM. However, if follow v2, it should be the original value. That's easy for user to find in the content.

[the remark this answers is not in the document]
Comment (mingwli) [reply docs/specs/toc.yml]: @yufeih @fenxu it's confirmed with Curt, we should leave the `{filePath}` as it is(what the writer has written), it's easier for them to locate the error.

**Pull request merged**